### PR TITLE
chore(Redesign Servizi): [IARS-28] Removes services redesign FF and cleanup dead code

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR removes the services redesign FF since the code should be persistent and removes unused code on services home component

### CODE REVIEW

**Review by pietro909 — APPROVED** [review]
tested locally, works just fine! pretty cool to see so much code go away 🚀

File: ts/components/services/NewServiceListItem.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ import { ProfileState } from "../../store/reducers/profile";
 import customVariables from "../../theme/variables";
 import { getEnabledChannelsForService } from "../../utils/profile";
 import ListItemComponent from "../screens/ListItemComponent";
-import { servicesRedesignEnabled } from "../../config";
 
 interface State {
   switchValue: boolean;
```
Comment (Undermaken) [inline ts/components/services/NewServiceListItem.tsx:13]: I guess we don't need anymore to handle the activation on list item. 
Could it be removed too?

File: ts/components/services/NewServiceListItem.tsx [hunk 2]
```diff
@@ -20,12 +19,6 @@ type Props = Readonly<{
   onSelect: (service: ServicePublic) => void;
   isRead: boolean;
```
Comment (Undermaken) [inline ts/components/services/NewServiceListItem.tsx:20]:
```suggestion
  isRead: boolean;
```
Same of above?

File: ts/components/services/ServicesTab.tsx [hunk 3]
```diff
@@ -216,9 +166,7 @@ class ServicesTab extends React.PureComponent<Props> {
     // - if long press is enabled: a switch to enable/disable all related services
     // - if the organization is local a button to remove it
     // - none
```
Comment (Undermaken) [inline ts/components/services/ServicesTab.tsx:168]: I guess it could be removed

File: ts/components/services/ServicesTab.tsx [hunk 4]
```diff
@@ -240,9 +188,6 @@ class ServicesTab extends React.PureComponent<Props> {
             ? new Set(this.props.selectedOrganizations || [])
             : undefined
         }
-        onLongPressItem={this.props.handleOnLongPressItem}
-        isLongPressEnabled={this.props.isLongPressEnabled}
-        onItemSwitchValueChanged={this.props.onItemSwitchValueChanged}
         animated={this.onTabScroll()}
         renderRightIcon={renderRightIcon}
```
Comment (Undermaken) [inline ts/components/services/ServicesTab.tsx:192]: Do we still need it?
If no, it could be removed

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Comment by Undermaken** [comment]
@CrisTofani I took the opportunity to remove other thing about national/local and long press features
Could you check this commit https://github.com/pagopa/io-app/pull/3220/commits/1310e06852cf0b95cf755bb51533024e7fb59f93?
If it has some errors I can revert it

**Comment by CrisTofani** [comment]
> @CrisTofani I took the opportunity to remove other things about **_national/local_**, **_read_** state and **_long press_** features
> Could you check this commit [1310e06](https://github.com/pagopa/io-app/commit/1310e06852cf0b95cf755bb51533024e7fb59f93)?
> If it has some errors I can revert it

@Undermaken starting from your cleanup I went a little bit further cleaning up that code from components, can you give one last check?

**Comment by Undermaken** [comment]
> > @CrisTofani I took the opportunity to remove other things about **_national/local_**, **_read_** state and **_long press_** features
> > Could you check this commit [1310e06](https://github.com/pagopa/io-app/commit/1310e06852cf0b95cf755bb51533024e7fb59f93)?
> > If it has some errors I can revert it
> 
> @Undermaken starting from your cleanup I went a little bit further cleaning up that code from components, can you give one last check to the commit [abbcd68](https://github.com/pagopa/io-app/commit/abbcd6871338fde0e1d65aeafc225d53e9d4e157)?

It looks good! thank you
I removed other minor stuffs here https://github.com/pagopa/io-app/pull/3220/commits/79895c650e5f0926ede8e344843c83f65a574879 and I should also fixed a compile error introduced on your last commit
Please double check

**Comment by CrisTofani** [comment]
@Undermaken lgtm

**Review by Undermaken — APPROVED** [review]

File: ts/components/services/NewServiceListItem.tsx [hunk 5]
```diff
@@ -51,99 +35,27 @@ const styles = StyleSheet.create({
...
+  const potService = props.item;
+  const onPress = pot.toUndefined(
+    pot.map(potService, service => () => props.onSelect(service))
+  );
+
+  const serviceName = pot.isLoading(potService)
+    ? I18n.t("global.remoteStates.loading")
+    : pot.isError(potService) || pot.isNone(potService)
+    ? I18n.t("global.remoteStates.notAvailable")
+    : potService.value.service_name;
```
Comment (pietro909) [inline ts/components/services/NewServiceListItem.tsx:48]: with 4 branches combine in ternaries, I would have preferred a `fold` operation tbh 🤔

**Review by pietro909 — APPROVED** [review]
Tested locally on IOS.

File: ts/components/services/NewServiceListItem.tsx (hunk 5 above)
Comment (Undermaken) [reply ts/components/services/NewServiceListItem.tsx]: Good point. Me too https://github.com/pagopa/io-app/pull/3220/commits/5cc0a639c2cbcfd0b1e2f2c530c80ec2aee0e36d

Comment (CrisTofani) [reply ts/components/services/NewServiceListItem.tsx]: more readable fro sure, even if I don't really appreciate the `pot.fold` syntax 😆

**Review by CrisTofani — COMMENTED** [review]
lgtm

**Pull request merged**